client/src/pages/AddTaskPage.tsx

```diff
@@ -32,51 +24,19 @@ export const AddTaskPage: React.FC<RouteComponentProps> = ({ history, match }) =
     setShowToast(true);
   };
 
-  const submit = (event: SyntheticEvent) => {
-    event.preventDefault();
-    const variables = {
-      input: {
-        title,
-        description,
-        status: 'OPEN',
-        version: 1
-      },
-    };
-    
+  const submit = (model: any) => {    
     createTaskMutation({
-      variables
+      variables: {input: {...model},}
     })
-    .then(() => history.push('/'))
-    .catch(handleError);
+      .then(() => history.push('/'))
+      .catch(handleError);
   };
   
   return (
     <>
       <Header title="Add task" backHref="/tasks" match={match} />
       <IonContent>
-        <form onSubmit={submit} style={{ padding: '0 16px' }}>
-          <IonItem>
-            <IonLabel color="primary" position="floating">Title</IonLabel>
-            <IonInput 
-              type="text" 
-              required 
-              name="title" 
-              value={title} 
-              onInput={(e:any) => setTitle(e.target.value)} 
-            />
-          </IonItem>
-          <IonItem>
-            <IonLabel color="primary" position="floating">Description</IonLabel>
-            <IonInput 
-              type="text" 
-              required 
-              name="description" 
-              value={description} 
-              onInput={(e:any) => setDescription(e.target.value)}
-            />
-          </IonItem>
-          <IonButton className="submit-btn" expand="block" type="submit">Create</IonButton>
-        </form>
+        <TaskForm handleSubmit={submit} model={{}} />
```

Comment:
An empty object is the default value for `model`.

Reply:
Yeah, I'm overloading the TaskForm component for update and edit so I'm passing an empty object in for the `add component` and then passing in values for the `edit/update component`, which allows me to re-use the form